CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+## main
```

Comment:
Just because of the addition of `final` or are there other changes you would consider breaking? We need to avoid breaking changes right now, so please list anything you think is breaking and then let's discuss.

Reply:
I suppose new `final`'s were the only reason. As we discussed that topic already, we can close this discussion.